InriaMuseum/InriaMuseum.html

```diff
@@ -6,6 +6,8 @@
     <script>
         var current_page; //global variable
         var previous_page;
+        var map_check = 0; // if map_check = 1 execute change_map()
```

Comment:
great changes! Quick suggestion: replace 1 and 0 by `true` and `false`, and rename the variable to `f_call_change_map`. By convention, variables starting with "f" means "flag", which indicates it's a boolean that, if set to 1, means or triggers something